core/__tests__/bin/push.ts

```diff
@@ -182,6 +182,94 @@ describe("bin/push", () => {
     );
   });
 
+  test("can retry if there is a timeout reached", async () => {
+    const timeoutNock = nock("https://cloud.grouparoo.com")
+      .post("/api/v1/configuration")
+      .query(true)
+      .times(4)
+      .replyWithError({ message: "timeout!", code: "ETIMEDOUT" });
+
+    const happyNock = nock("https://cloud.grouparoo.com")
+      .post("/api/v1/configuration")
+      .query(true)
+      .reply(200, {
+        configuration: { id: "config-id" },
+      });
+
+    nock("https://cloud.grouparoo.com")
+      .get("/api/v1/configuration/config-id")
+      .query(true)
+      .reply(200, {
+        configuration: {
+          validateJobId: "validate-id",
+          state: "validated",
+        },
+      });
+
+    nock("https://cloud.grouparoo.com")
+      .get("/api/v1/configuration/config-id")
+      .query(true)
+      .reply(200, {
+        configuration: {
+          state: "finished",
+        },
+      });
+
+    nock("https://cloud.grouparoo.com")
+      .get("/api/v1/job/validate-id")
+      .query(true)
+      .reply(200, {
+        job: {
+          type: "validate",
+          logs: "validate job logs go here",
+        },
+      });
+
+    const command = new Push();
+    const toStop = await command.run({
+      params: {
+        archivePath,
+        projectId: "my-project",
+        token: "some-token",
+        apply: false,
+      },
+    });
+    expect(toStop).toBe(true);
+
+    expect(timeoutNock.isDone()).toBe(true);
+    expect(happyNock.isDone()).toBe(true);
+
+    const output = messages.join(" ");
+    console.log(output);
```

Comment:
remove log?

Reply:
whoops! the log in that test was mocked so I didn't see it :D